Dockerfile

```diff
@@ -2,6 +2,12 @@ FROM python:3.9
 ADD . /src
 WORKDIR /src
 
+RUN pip install --upgrade pip
+
+RUN pip install --no-cache-dir newrelic
+
+RUN pip install futures==3.1.1
```

Comment:
This line is an artifact of some dependency issues that got resolved last sprint, we can get rid of it! (my bad here)